### ISSUE TITLE
Idea for visualizing size

### ISSUE THREAD

**Issue by parbo** [google/bloaty#85]
The tools/binary_size/supersize tool from chromium can produce things like this: 

https://agrieve.github.io/chrome/supersize-chrome-android-oct-19-2017/

Something like that would be very useful!

**Comment by prusnak** [comment]
Sort-of duplicate of https://github.com/google/bloaty/issues/53

**Comment by butterl** [comment]
Good idea, also looking for this kind of  visualization way to monitor the memory growth :+1:

**Comment by RoboGnome** [comment]
I have gotten some pretty good results with microsofts sanddance: https://www.microsoft.com/en-us/research/project/sanddance/ 

I could create the memory layout per section for the whole project or for each compile unit individually.
It was also straight forward to color the memory map by symbol, showing a a quick representation of the biggest symbols.
Here is an example from a small test project:
![image](https://user-images.githubusercontent.com/26966675/147382437-ebf749b9-9309-43b0-91b0-49c99b0cd407.png)

![image](https://user-images.githubusercontent.com/26966675/147382448-0813688e-0829-43e1-80a8-6d52f3ff9026.png)
